src/App.vue

```diff
@@ -3,10 +3,6 @@
     <b-navbar>
       <template slot="start">
         <b-navbar-item tag="router-link" :to="{ path: '/' }">Home</b-navbar-item>
-        <b-navbar-item
```

Comment:
was unused, game reference was old schema - mainly used this for testing.

TODO: add a list of games that user is associated to